### PR TITLE
[mailtrain] Create guide

### PR DESCRIPTION
**Pull request by felixfoertsch**
#430 

I wrote the guide based on my recent setup. Please review and merge. 👏

### CODE REVIEW

File: source/guide_mailtrain.rst [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+In the **Mailer Settings** section change the
+
+  * *Hostname* to ``stardust.uberspace.de``,
+  * *Port* to ``587``,
+  * *Encryption* to ``Use STARTTLS - usually selected for port 587 and 25``,
+  * *username* to ``isabell@uber.space``,
+  * *password* to ``MySuperSecretPassword`` and
+  * test your settings by pressing the Button **Check Mailer Config**.
+
+.. note:: Uberspace discourages **mass** mailings. If you regularly send a large amount of emails, consider using `AWS SES <https://aws.amazon.com/ses/>`_ instead of the Uberspace mailing system.
```
Comment (luto) [inline source/guide_mailtrain.rst:158]: Our [house rules](https://dashboard.uberspace.de/register?lang=en) state:

> Mass mailings such as newsletters are not allowed.

You're free to use your uberspace as an admin interface, though. Please remove the uberspace SMTP settings from the guide and add a note telling users that they should not use them. If possible, suggest a few other services like SES, Mailjet, SendGrid, ...

Comment (felixfoertsch) [reply source/guide_mailtrain.rst]: Pushed the updated version.

**Comment by luto** [comment]
Thank you! :tada:

**Comment by felixfoertsch** [comment]
Woohoo! My first guide in the new lab! Thanks @luto

**Pull request merged**